egs/dirha_wsj/asr1/local/tools/create_folder_str.m

```diff
@@ -0,0 +1,30 @@
+% This script generate a folder (folder_new) with the same strcuture of the input folder (folder_old) 
```

Comment:
no copyright and license...
We should be careful when we use a such file.

Reply:
All the matlab files were directly used from https://github.com/SHINE-FBK/DIRHA_English_wsj where the original versions have no copyright or license. Shall we clarify in each individual matlab files?